modules/ROOT/pages/legacy-search-experience.adoc

```diff
@@ -0,0 +1,311 @@
+= Legacy Search Experience
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+The legacy search experience, which was previously called the old search, is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. This search experience does not include enhanced options for searching by rank, organizations, and lifecycle states. 
+
+To access the legacy search, click *Back to legacy search* from the landing page.
+
+Discover assets by searching for specific assets from the landing page of the Exchange catalog. 
+
+== Display Assets in List or Grid View
+
+You have an option to display the catalog of assets in list or grid view. List view arranges assets in a list format and grid view displays the assets in tiles. Both views display the asset name, type, rating, and created by information.
+
+== Filter
+
+You can filter search results by asset types for example, API groups, REST APIs, and rulesets by selecting an option from the *All types* drop-down menu. The options that appear depend on which types are available within an organization.
```

Comment:
The first sentence feels like a lot. Maybe parenthetical? 

You can filter search results by asset types (for example, API groups, REST APIs, and rulesets) by selecting an option from the *All types* drop-down menu.

Reply:
fixed